libs/librrgraph/src/base/rr_graph_view.h

```diff
@@ -1,36 +1,42 @@
 #ifndef RR_GRAPH_VIEW_H
 #define RR_GRAPH_VIEW_H
 
-#include "rr_graph_builder.h"
-#include "rr_node.h"
-#include "physical_types.h"
-/* An read-only routing resource graph
- * which is an unified object including pointors to
- * - node storage
- * - TODO: edge_storage
- * - TODO: node_ptc_storage
- * - TODO: node_fan_in_storage
- * - rr_node_indices
+/**
+ * @file
+ * @brief The RRGraphView encapsulates a read-only routing resource graph, providing clients with tailored frame views of the object. 
```

Comment:
I'm not quite sure what "providing clients with tailored frame views of the object." means.  I think this may be something that is hoped for in the future but isn't there now.  I suggest saying:
" ... a read-only routing resource graph as most clients (router, timing analyzer, etc.) only need to read a routing-resource graph."